### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by shkoder**

### CODE REVIEW

**Comment by lightbend-cla-validator** [comment]
Hi @shkoder,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

File: src/main/g8/build.sbt [hunk 1]
```diff
@@ -12,7 +11,6 @@ lazy val root = (project in file(".")).
       "com.typesafe.akka" %% "akka-http"            % akkaHttpVersion,
       "com.typesafe.akka" %% "akka-http-spray-json" % akkaHttpVersion,
       "com.typesafe.akka" %% "akka-http-xml"        % akkaHttpVersion,
-      "com.typesafe.akka" %% "akka-stream"          % akkaVersion,
```
Comment (jlprat) [inline src/main/g8/build.sbt]: Better to not remove this one. It is explicitly set to specify that we want to use the a different Akka Version which is not the one bundled with Akka HTTP.

File: src/main/g8/default.properties [hunk 2]
```diff
@@ -1,5 +1,4 @@
 name=My Akka HTTP Project
-scala_version=2.12.2
+scala_version=2.12.3
 akka_http_version=10.0.9
-akka_version=2.5.3
```
Comment (jlprat) [inline src/main/g8/default.properties]: Same as above

File: src/main/g8/src/main/scala/com/lightbend/akka/http/sample/JsonSupport.scala [hunk 3]
```diff
@@ -2,7 +2,7 @@ package com.lightbend.akka.http.sample
 
 import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport
 import com.lightbend.akka.http.sample.UserRegistryActor.ActionPerformed
-import spray.json.{ DefaultJsonProtocol, RootJsonFormat }
+import spray.json.DefaultJsonProtocol
```
Comment (jlprat) [inline src/main/g8/src/main/scala/com/lightbend/akka/http/sample/JsonSupport.scala]: Well spotted :)

File: README.md [hunk 4]
```diff
@@ -13,33 +13,19 @@ This template will prompt for the following parameters. Press `Enter` if the def
...
 - `organization`: Specifies the organization for this project.
 
-This template comes with 2 different ways to implement an Akka HTTP server, along with their respective tests.
+The template comes with the following sources:
 
-- WebServer: This is the manual way to start a server. For this case, routes are defined in separated classes: the object `com.example.routes.BaseRoutes`
-and the trait `com.example.routes.SimpleRoutes`.
-- WebServerHttpApp: This server is started using the recently introduced `HttpApp`, which bootstraps a server with just a few lines.
-For this particular case, the routes are defined in the same class, but you can also define them in separated traits or objects.
+* `QuickstartServer.scala` -- contains the main class which sets-up and all actors, it runs the Akka HTTP `routes`.
```
Comment (jlprat) [inline README.md]: Maybe specify that this is the place where the routes are wrapped up together?

File: src/main/g8/build.sbt (hunk 1 above)
Comment (shkoder) [inline src/main/g8/build.sbt]: reverted, the concern is with
```
[warn] There may be incompatibilities among your library dependencies.
[warn] Here are some of the libraries that were evicted:
[warn]     * com.typesafe.akka:akka-stream_2.12:2.4.19 -> 2.5.3
[warn] Run 'evicted' to see detailed eviction warnings
```
setting akka_version=2.4.19 in default.properties would get rid of the warning

**Comment by jonas** [comment]
Fixes #17

File: src/main/g8/build.sbt (hunk 1 above)
Comment (jlprat) [inline src/main/g8/build.sbt]: That warning is expected, and it's not a problem as Akka 2.4.19 and 2.5.3 are binary compatible.

Comment (shkoder) [inline src/main/g8/build.sbt]: ok

**Pull request merged**